Serializer/Normalizer/FormErrorHandler.php

```diff
@@ -86,6 +90,11 @@ public function serializeFormToYml(YamlSerializationVisitor $visitor, Form $form
         return $result;
     }
 
+    public function __call($name, $arguments)
+    {
+        return call_user_func_array([$this->formErrorHandler, $name], $arguments);
```

Comment:
Is this for bc? If so I think we should deprecate these calls.

Reply:
I decided to do this since the `FormErrorHandler` from JMSSerializerBundle does not implement an interface and I don't want to us to have to update our class every time a change in JMSSerializerBundle is made (i.e. new methods are added).